speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DeveloperMenu.cpp

```diff
@@ -265,6 +276,43 @@ static void SetSteerThreshold(void*)
     SetThreshold(m_decisionThresholds.Steer, m_steerThresholdControl);
 }
 
+/// @breif set the default values of threshold boxes based on the InterventionType.
+static void SetDefaultValues(void*)
+{
+    switch (SMediator::GetInstance()->GetInterventionType())
+    {
+        case INTERVENTION_TYPE_ONLY_SIGNALS:
+        {
+            // set default threshold values for only signals
+            m_decisionThresholds.Accel = 1;
```

Comment:
Preferably put these defaults in a define macro at the top of the file instead of hidden in a function